src/wget.c

```diff
@@ -1218,7 +1218,10 @@ static int process_response_header(wget_http_response_t *resp)
 	DOWNLOADER *downloader = job->downloader;
 	wget_iri_t *iri = job->iri;
 
-	print_status(downloader, "HTTP response %d %s\n", resp->code, resp->reason);
+	if(resp->code == 200 || resp->code == 301 || resp->code == 302 || resp->code == 304)
```

Comment:
I don't think this is a very good idea. Handling individual response codes will soon get too cumbersome. Currently, this treats a 206 Partial Content as an `ERROR` as well. 

Pretty much only the 4xx and 5xx series of responses are errors (Client and Server, respectively). If you'd really like this in the output, please match on the series instead of individual response code values